specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-11-15-preview/managedCassandra.json

```diff
@@ -774,657 +693,1717 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          },
+          "202": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CassandraReaperClusterStatus"
+            }
           }
         }
       }
     },
-    "ManagedCassandraARMResourceProperties": {
-      "type": "object",
-      "description": "The core properties of ARM resources.",
-      "properties": {
-        "id": {
-          "readOnly": true,
-          "type": "string",
-          "description": "The unique resource identifier of the ARM resource."
-        },
-        "name": {
-          "readOnly": true,
-          "type": "string",
-          "description": "The name of the ARM resource."
-        },
-        "type": {
-          "readOnly": true,
-          "type": "string",
-          "description": "The type of Azure resource."
-        },
-        "location": {
-          "type": "string",
-          "description": "The location of the resource group to which the resource belongs."
-        },
-        "tags": {
-          "$ref": "cosmos-db.json#/definitions/Tags"
-        },
-        "identity": {
-          "$ref": "#/definitions/ManagedCassandraManagedServiceIdentity"
-        }
-      },
-      "x-ms-azure-resource": true
-    },
-    "ManagedCassandraManagedServiceIdentity": {
-      "description": "Identity for the resource.",
-      "type": "object",
-      "properties": {
-        "principalId": {
-          "readOnly": true,
-          "type": "string",
-          "description": "The object id of the identity resource."
-        },
-        "tenantId": {
-          "readOnly": true,
-          "type": "string",
-          "description": "The tenant id of the resource."
-        },
-        "type": {
-          "type": "string",
-          "description": "The type of the resource.",
-          "enum": [
-            "SystemAssigned",
-            "None"
-          ],
-          "x-ms-enum": {
-            "name": "ManagedCassandraResourceIdentityType",
-            "modelAsString": true
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/cassandraClusters/{clusterName}/repairTables": {
+      "post": {
+        "operationId": "CassandraRepair_GetTableStatus",
```

Comment:
Please think more carefully about naming the action and the operation id. Is it the 'repair the tables' action, or is it the 'get table repair status' action, or is it the 'list table repair operations action'?